src/components/AddItem/AddItem.style.scss

```diff
@@ -62,3 +86,99 @@
     }
   }
 }
+
+// Modal Icon
+
+#list {
+  &__icon {
+    width: 1.6em;
+    height: 1.6em;
+  }
+}
+
+// Modal Styling
+
+.list {
+  &__modal {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__card {
+    padding: 1rem;
+    max-width: 550px;
+    box-shadow: -1px -2px 28px 0px rgba(0, 0, 0, 0.75);
+    background-color: whitesmoke;
+    border-radius: 4px;
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+@media screen and (max-width: 620px) {
+  // Overall page layout
+  .page {
+    &__container {
+      width: 100vw;
+      height: 100%;
+    }
+    &__content {
+      width: 100vw;
+      height: calc(100vh + 10px);
+      border-radius: inherit;
+      box-shadow: none;
+    }
+  }
+
+  // "Tooltip" styling
+
+  .tooltip {
+    margin: 0.5rem 0;
+  }
+
+  .item {
+    &__wrapper {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+
+    &__form {
+      margin-bottom: 1rem;
+
+      &__wrapper {
+        width: 100%;
+      }
+    }
+  }
+
+  .button {
+    &__container {
+      padding-top: 1rem;
+    }
+  }
+
+  // modal styling
+  .list {
+    &__card {
+      max-width: 90vw;
+    }
+  }
+}
+
+// iPhone 5/SE specific layout
```

Comment:
As a former SE user for many years, I really appreciate seeing this!

Reply:
Hahaha, dude this bit had me being like "...do i REALLY got to do this rn"